openapi/single-sign-on/single-sign-on.yaml

```diff
@@ -0,0 +1,384 @@
+          description: 'When using `authorization_code` grant: the exact `redirect_uri` that was used at the authorization endpoint'
+        - schema:
+            type: string
+          in: query
+          name: client_id
+          description: 'When using `authorization_code` or `refresh_token` grant: The client ID for the authorization flow.'
+        - schema:
+            type: string
+          in: query
+          name: client_secret
+          description: 'When using `authorization_code` or `refresh_token` grant: The client secret for the authorization flow. Required when obtaining refresh tokens or using them as grants.'
+        - schema:
+            type: string
+          in: query
+          name: scope
+          description: 'When using `...jwt-bearer` (assertion) or `refresh_token` grant:  The scopes to be used rather than scope from assertion or from previous refresh token. `offline_access` scope may not be used by assertion grants, and scopes can only be permanently narrowed [check?] for refresh token grants.'
+        - schema:
+            type: string
+          in: query
+          name: code_verifier
+          description: 'When using `authorization_code`: The PKCE code verifier for the `code_challenge` presented to the authorization endpoint starting this flow. This is used to protect against security attacks and should be used by all, but [is required for public clients].'
+      tags:
+        - Single-Sign-On
+      security: []
+  /oauth2/user-info:
+    get:
+      summary: OpenID User Info
+      operationId: openid-userinfo
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                x-examples:
+                  example-1:
+                    sub: '248289761001'
+                    name: Jane Doe
+                    given_name: Jane
+                    family_name: Doe
+                    preferred_username: j.doe
+                    email: janedoe@example.com
+                    picture: 'http://example.com/janedoe/me.jpg'
+                properties:
+                  sub:
+                    type: string
+                    minLength: 1
+                  name:
+                    type: string
+                    minLength: 1
+                  given_name:
+                    type: string
+                    minLength: 1
+                  family_name:
+                    type: string
+                    minLength: 1
+                  preferred_username:
+                    type: string
+                    minLength: 1
+                  email:
+                    type: string
+                    minLength: 1
+                  picture:
+                    type: string
+                    minLength: 1
+                  namespace:
+                    type: string
+                  product_navbar_data_url:
```

Comment:
This field is one that deserves extra attention/explanation but doesnt have it right now